docs/contributing.rst

```diff
@@ -184,8 +184,10 @@ To release a new version of the theme, core team will take the following steps:
 #. Update the changelog (``docs/changelog.rst``) with the version information.
 #. Run ``python setup.py update_translations`` to compile new translation files
    and update Transifex.
-#. Run ``python setup.py build_assets`` to rebuild all the theme assets and the Python
-   package.
+#. Run ``npm run build`` to rebuild all the theme assets and the Python
+   package, notice that ``package-lock.json`` will be updated with a new package
```

Comment:
`package-lock.json` is updated on `npm install`, not `npm run build`. `npm install` might run if you've never run `npm run build` before though.

Reply:
True. Shouldn't we just add `npm run install` then?

Reply:
Perhaps it is better to exclude anything that touches `package-lock.json` from the build process.

Reply:
I think the existing commands using `python setup.py build_assets` do all of this for us. It doesn't seem like we have any strong reason to change this, really.

> Perhaps it is better to exclude anything that touches package-lock.json from the build process.

From the release process that is? It's best to have the lock file represent the build artifacts probably, I think it's helpful to have included in the release. It's not likely to be of much use to anyone, but will be used in CI to verify the release (as dependencies will install via `npm ci` in CI).

Reply:
This might be a Catch 22... what I mean is that *if* package-lock.json isn't touched during the release process, we shouldn't make updates to it.*If* we need to run commands that changes it, we should also be aware.